team-norms/bau_expectations.md

```diff
@@ -0,0 +1,20 @@
+# Business As Usual (BAU) Expectations
+
+Teams must deliver software systems in accordance with our Business As Usual Expectations that make software systems easier to run, debug, and improve.
+
+The Made Tech Support Team requirements are identified below. Where the application is transitioning to a customer support team, you should liaise with the customer team to understand their specific needs.
+
+## Application Requirements
+
+- Centralised Logging (Papertrail)
+- Application Performance Monitoring (Datadog)
+- Exception Tracking (Sentry)
+- Automated Dependency Upgrades (Dependabot)
+- Automated Security Patches (Snyk)
```

Comment:
Possibility we might be able to use Github security alerts for this now, which wouldn't require an additional charge. Needs testing though how good it actually is, if at all.

Reply:
OK, what do you want to do in here? Leave blank to force conversation with team?